src/Comments.tsx

```diff
@@ -14,23 +14,38 @@ interface CommentsResponse {
 
 const Comments = () => {
   const [comments, setComments] = useState<Comment[]>([]);
+  const [isDataLoading, setIsDataLoading] = useState(false);
```

Comment:
probably the initial value should be true?
```suggestion
  const [isDataLoading, setIsDataLoading] = useState(true);
```

Reply:
Yes. Thanks.